packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -185,6 +185,12 @@ exports.createSchemaCustomization = ({ actions, schema }) => {
           }),
         },
         navLevels: { type: 'Int!' },
+        timeToRead: {
```

Comment:
* Why does the GraphQL facade have to pass through both values and not just the effective time to read to be displayed?  The point of introducing a facade is to hide the business logic behind it and not to expose it. I.e. the logic "if the manual frontmatter is set, use it, otherwise use the automatic value" belongs here and not into the frontend code. 
* IMHO, if not necessary for a reasons, the resolverPassthrough should be avoided - it's the thing that makes the dev server crash if something is not OK in the source node.